dev/io.openliberty.org.apache.myfaces.4.0_fat/fat/src/io/openliberty/org/apache/myfaces40/fat/tests/MultipleInputFileTest.java

```diff
@@ -0,0 +1,355 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.org.apache.myfaces40.fat.tests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Iterator;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+import org.xml.sax.helpers.AttributesImpl;
+
+import com.gargoylesoftware.htmlunit.NicelyResynchronizingAjaxController;
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.DomElement;
+import com.gargoylesoftware.htmlunit.html.HtmlElement;
+import com.gargoylesoftware.htmlunit.html.HtmlFileInput;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import com.gargoylesoftware.htmlunit.html.parser.neko.HtmlUnitNekoHtmlParser;
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.HttpUtils;
+import io.openliberty.org.apache.myfaces40.fat.FATSuite;
+
+/**
+ * Tests for the <h:inputFile/> with multiple attribute which is a new feature for Faces 4.0.
+ *
+ * <pre>
+ * Testing matrix:
+ *
+ * | Multiple Attribute | Async Javascript | Number of files |
+ * | ================== | ================ | =============== |
+ * |      UNSET         |      false       |      single     |
+ * |      UNSET         |      false       |     multiple    |
+ * |    "multiple"      |      false       |      single     |
+ * |    "multiple"      |      false       |     multiple    |
+ * |      UNSET         |      true        |      single     |
+ * |      UNSET         |      true        |     multiple    |
+ * |    "multiple"      |      true        |      single     |
+ * |    "multiple"      |      true        |     multiple    |
+ *
+ * </pre>
+ */
+@RunWith(FATRunner.class)
+public class MultipleInputFileTest {
```

Comment:
Have you given any thought to adding one or more of these tests to the FULL bucket execution vs LITE?